interface/skin.dmf

```diff
@@ -1,880 +1,923 @@
+		anchor1 = none
+		anchor2 = none
+		background-color = none
 		is-visible = false
+		saved-params = ""
 	elem "tooltip"
 		type = BROWSER
 		pos = 0,0
 		size = 999x999
+		anchor1 = none
+		anchor2 = none
+		background-color = none
 		is-visible = false
-
+		saved-params = ""
 
 window "mapwindow"
 	elem "mapwindow"
 		type = MAIN
-		is-pane = true
+		pos = 281,0
+		size = 640x480
+		anchor1 = none
+		anchor2 = none
+		background-color = none
 		saved-params = "pos;size;is-minimized;is-maximized"
+		is-pane = true
 	elem "map"
 		type = MAP
-		is-default = true
+		pos = 0,0
 		size = 640x480
 		anchor1 = 0,0
 		anchor2 = 100,100
 		font-family = "Arial"
 		font-size = 7
-		font-style = ""
 		text-color = none
-		background-color = none
-		icon-size = 0
+		is-default = true
+		drop-zone = true
 		saved-params = "icon-size"
 
-
 window "infowindow"
 	elem "infowindow"
 		type = MAIN
-		is-pane = true
+		pos = 281,0
+		size = 640x480
+		anchor1 = none
+		anchor2 = none
+		background-color = none
 		saved-params = "pos;size;is-minimized;is-maximized"
+		is-pane = true
 	elem "info"
 		type = CHILD
 		pos = 0,30
 		size = 640x445
 		anchor1 = 0,0
 		anchor2 = 100,100
-		is-vert = false
-		splitter = 50
-		show-splitter = true
+		background-color = none
+		saved-params = "splitter"
 		left = "statwindow"
 		right = "outputwindow"
-		saved-params = "splitter"
-  elem "changelog"
-		type = BUTTON
-		button-type = pushbutton
-		pos = 16,5
-		size = 104x20
-		anchor1 = 3,0
-		anchor2 = 19,0
-		text = "Changelog"
-		command = "changelog"
+		is-vert = false
 	elem "rules"
 		type = BUTTON
-		button-type = pushbutton
-		pos = 120,5
-		size = 100x20
+		pos = 186,4
+		size = 89x20
 		anchor1 = 19,0
 		anchor2 = 34,0
+		background-color = none
+		saved-params = "is-checked"
 		text = "Rules"
 		command = "rules"
 	elem "wiki"
 		type = BUTTON
-		button-type = pushbutton
-		pos = 220,5
-		size = 100x20
+		pos = 274,4
+		size = 89x20
 		anchor1 = 34,0
 		anchor2 = 50,0
+		background-color = none
+		saved-params = "is-checked"
 		text = "Wiki"
 		command = "wiki"
 	elem "forum"
 		type = BUTTON
-		button-type = pushbutton
-		pos = 320,5
-		size = 100x20
+		pos = 363,4
+		size = 89x20
 		anchor1 = 50,0
 		anchor2 = 66,0
+		background-color = none
+		saved-params = "is-checked"
 		text = "Forum"
 		command = "forum"
-  elem "github"
-	  type = BUTTON
-	  button-type = pushbutton
-		pos = 420,5
-		size = 100x20
+	elem "github"
+		type = BUTTON
+		pos = 452,4
+		size = 89x20
 		anchor1 = 66,0
 		anchor2 = 81,0
-	  text = "Github"
-	  command = "github"
-  elem "report-issue"
-	  type = BUTTON
-	  button-type = pushbutton
-		pos = 520,5
-		size = 100x20
+		background-color = none
+		saved-params = "is-checked"
+		text = "Github"
+		command = "github"
+	elem "set-fps"
+		type = BUTTON
+		pos = 10,4
+		size = 89x20
+		anchor1 = none
+		anchor2 = none
+		background-color = #e9f4f8
+		saved-params = "is-checked"
+		text = "Set FPS"
+		command = "fpssetting"

```

Comment:
i will this was a hack job done by the CIA to rush it out to show MSO before he passes out